examples/petsc/bps.c

```diff
@@ -112,6 +116,48 @@ struct User_ {
   Ceed ceed;
 };
 
+typedef enum {
+  CEED_BP1, CEED_BP2, CEED_BP3, CEED_BP4, CEED_BP5, CEED_BP6
+} bpType;
+static const char *const bpOptions[] = {"bp1","bp2","bp3","bp4","bp5","bp6",
```

Comment:
PETSc convention would be to call these `*Types` rather than `*Options`.

Reply:
Fixed